--- httpd.conf.default	2015-03-20 01:23:09.310841200 +0900
+++ httpd.conf.template	2015-03-20 01:23:29.535293300 +0900
@@ -67,7 +67,7 @@
 #
 # Timeout: The number of seconds before receives and sends time out.
 #
-Timeout 60
+Timeout	#{ de_deeeen[0:1200] }
 
 #
 # KeepAlive: Whether or not to allow persistent connections (more than
@@ -80,13 +80,13 @@
 # during a persistent connection. Set to 0 to allow an unlimited amount.
 # We recommend you leave this number high, for maximum performance.
 #
-MaxKeepAliveRequests 100
+MaxKeepAliveRequests	#{ keep_a_live[0:1000] }
 
 #
 # KeepAliveTimeout: Number of seconds to wait for the next request from the
 # same client on the same connection.
 #
-KeepAliveTimeout 15
+KeepAliveTimeout	#{ keep_de_deeen[0:50] }
 
 ##
 ## Server-Pool Size Regulation (MPM specific)
@@ -100,12 +100,12 @@
 # MaxClients: maximum number of server processes allowed to start
 # MaxRequestsPerChild: maximum number of requests a server process serves
 <IfModule prefork.c>
-StartServers       8
-MinSpareServers    5
-MaxSpareServers   20
-ServerLimit      256
-MaxClients       256
-MaxRequestsPerChild  4000
+StartServers        #{ s_servers[0:50] }
+MinSpareServers     #{ min_spare_sv[0:10] }
+MaxSpareServers     #{ max_sp_sv[0:15] }
+ServerLimit         #{ sv_lmt[0:512] }
+MaxClients          #{ max_cli[0:512] }
+MaxRequestsPerChild	#{ uhehe[0:10000] }
 </IfModule>
 
 # worker MPM
